packages/common-utils/src/clickhouse.ts

```diff
@@ -458,7 +458,12 @@ export function chSqlToAliasMap(
       ast.columns.forEach(column => {
         if (column.as != null) {
           if (column.type === 'expr' && column.expr.type === 'column_ref') {
-            aliasMap[column.as] = column.expr.column.expr.value;
+            aliasMap[column.as] =
```

Comment:
this hot fix is good for now. I wonder if we could find how the node-sql-parser rebuilds the sql and use that function instead